lib/ansible/config/base.yml

```diff
@@ -613,6 +613,24 @@ DEFAULT_FORKS:
   ini:
   - {key: forks, section: defaults}
   type: integer
+DEFAULT_PROCESS_MODEL:
```

Comment:
DEFAULT_ is misnomer (originally true but not so since 0.2) I stopped using it a while back, we should really drop it from any new settings